league/manager/models.py

```diff
@@ -37,4 +68,4 @@ class Event(models.Model):
     time = models.DateTimeField()
     player = models.ForeignKey(TeamPlayer, on_delete=models.CASCADE)
     event_type = models.ForeignKey(EventType, on_delete=models.CASCADE)
-
+    match_id = models.ForeignKey(Match, on_delete=models.CASCADE)
```

Comment:
Please avoid adding _id suffixes - django adds them automatically - now you have match_id and match_id_id